adapters/smaato/image.go

```diff
@@ -0,0 +1,52 @@
+package smaato
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+)
+
+type imageAd struct {
+	Image image `json:"image"`
+}
+type image struct {
+	Img                img      `json:"img"`
+	Impressiontrackers []string `json:"impressiontrackers"`
+	Clicktrackers      []string `json:"clicktrackers"`
+}
+type img struct {
+	URL    string `json:"url"`
+	W      int    `json:"w"`
+	H      int    `json:"h"`
+	Ctaurl string `json:"ctaurl"`
```

Comment:
Does this have a friendlier variable name to use? I'm not sure what a CTA Url is.

Reply:
ctaurl translates to call_to_action destination URL.
it's a term used in OpenRTB spec

[OpenRTBSpec](https://iabtechlab.com/wp-content/uploads/2016/07/OpenRTB-Native-Ads-Specification-Final-1.2.pdf)

As ctaurl is something that is still being used internally in Smaato, We(Smaato) would prefer to keep it as so that the maintenance of the code be easier for us on a later date.

Reply:
Ok.